webapp/client/src/pages/NoPauschbetragPage.test.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import NoPauschbetragPage from "./NoPauschbetragPage";
+
+let props = {
+  stepHeader: {
+    title: "Title",
+    intro: "Intro",
+  },
+  prevUrl: "/some/prev/path",
+  nextUrl: "/some/next/path",
+};
+
+it("should render step header title", () => {
+  render(<NoPauschbetragPage {...props} />);
+  expect(screen.getByText("Title")).toBeInTheDocument();
+});
+
+it("should ignore step header intro", () => {
+  render(<NoPauschbetragPage {...props} />);
+  expect(screen.queryByText("Intro")).not.toBeInTheDocument();
+});
```

Comment:
I know that we have those tests in other places, too. I am wondering if we really need those tests? We are also not testing if they are ignoring other props. Wouldn't it be better to tests if the correct intro is set?

Reply:
Testing that the into is set is a bit senseless to me. But so might be testing this. I can totally delete this.